pkg/proc/fncall.go

```diff
@@ -363,6 +363,36 @@ func evalFunctionCall(scope *EvalScope, node *ast.CallExpr) (*Variable, error) {
 		if err != nil {
 			return nil, err
 		}
+	   case "ppc64le":
+                // debugCallV2 on arm64 needs a special call sequence, callOP can not be used
+                sp := regs.SP()
+                sp -= 4 * uint64(bi.Arch.PtrSize())
```

Comment:
Is this constant the only difference? Should we do `case "arm64", "ppc64le":` and then just an if to select this constant?

Reply:
Thanks for the suggestion have modified it accordingly